src/adapters/ws/server.ts

```diff
@@ -232,8 +232,8 @@ class WebSocketsAdapter extends Adapter {
       )
 
       const wsChannelBinding = this.parsedAsyncAPI
-        .channel(pathname)
-        .binding('ws')
+        .channels().get(pathname)
```

Comment:
Same as before. Pathname is not necessarily the name of the channel. It's the address for sure.